aiopg/utils.py

```diff
@@ -145,7 +146,7 @@ class _PoolAcquireContextManager(_ContextManager):
     __slots__ = ('_coro', '_conn', '_pool')
 
     def __init__(self, coro, pool):
-        self._coro = coro
+        super().__init__(coro)
```

Comment:
not sure about this one, constructor also sets _obj property, it is not listed in slots, probably better have original version

Reply:
actually, this seems more correct, `_obj` is used for `__aenter__` / `__aexit__` in the parent class, and this is overriding the behavior and thus should be using the same attribute

Reply:
with even a further simplification :)